### PR TITLE
New class created GiftCardTest

### PR DESCRIPTION
**Pull request by pbksoft2019**
can you please review my code

### CODE REVIEW

**Comment by avvaland26** [comment]
can u change

File: src/main/java/WebTestHomeWork/GiftCardTest.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package WebTestHomeWork;
+
+public class GiftCardTest {
+}
```
Comment (avvaland26) [inline src/main/java/WebTestHomeWork/GiftCardTest.java:4]: can u write code here?

**Review by avvaland26 — APPROVED** [review]

**Comment by pbksoft2019** [comment]
ok checked

File: src/main/java/WebTestHomeWork/GiftCardTest.java (hunk 1 above)
Comment (anjalipatel2422) [reply src/main/java/WebTestHomeWork/GiftCardTest.java:4]: get it

**Pull request merged**